### PR TITLE
Web console: honor cluster-resource-override-enabled

### LINKED ISSUE #7365: Web console should honor cluster resource override annotation
The web console should honor the `quota.openshift.io/cluster-resource-override-enabled` project annotation introduced in #6901.

/cc @sosiouxme @jwforres

### PR DESCRIPTION
**Pull request by spadgett**
Honor project annotation `quota.openshift.io/cluster-resource-override-enabled`.

Fixes #7365

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres PTAL

File: assets/app/scripts/services/limits.js [hunk 1]
```diff
@@ -33,14 +33,21 @@ angular.module("openshiftConsole")
...
+    var limitRequestOverridesEnabled = function(project) {
+      // If config is absent, overrides are disabled.
+      if (!LIMIT_REQUEST_OVERRIDES) {
+        return false;
+      }
+
+      // If the project resource override annotation is unset or is set to "true", overrides are enabled.
+      var projectOverrideAnnotation =
+        $filter('annotation')(project, 'quota.openshift.io/cluster-resource-override-enabled');
+      return !projectOverrideAnnotation || projectOverrideAnnotation === 'true';
```
Comment (spadgett) [inline assets/app/scripts/services/limits.js]: @sosiouxme please check this behavior: overrides are enabled if config is set AND project annotation is missing or set to the string `"true"`.

Comment (sosiouxme) [inline assets/app/scripts/services/limits.js]: @spadgett confirmed

**Comment by jwforres** [comment]
[merge]

**Pull request merged**